app/models/notifier.rb

```diff
@@ -168,12 +170,24 @@ def changeset_comment_notification(comment, recipient)
                   I18n.t("notifier.changeset_comment_notification.commented.subject_other", :commenter => @commenter)
                 end
 
+      attachments.inline["logo.png"] = File.read(Rails.root.join("app", "assets", "images", "osm_logo_30.png"))
```

Comment:
Shouldn't thin use `image_path` or something? Explicitly building a path to an asset like that seems wrong.